source/integrations/index.html

```diff
@@ -210,6 +213,12 @@
             return comp.featured;
           };
 
+        } else if (hash === '#works-with-home-assistant' || hash === '') {
```

Comment:
`|| hash === ''` can be removed as the previous if statement (featured) has it already and therefore, we would never come to this point if `hash === ''`